packages/nodes-base/nodes/Telegram/Telegram.node.ts

```diff
@@ -167,11 +173,13 @@ export class Telegram implements INodeType {
 						name: 'Answer Query',
 						value: 'answerQuery',
 						description: 'Send answer to callback query sent from inline keyboard',
+						action: 'Answer Query a callback',
 					},
 					{
 						name: 'Answer Inline Query',
 						value: 'answerInlineQuery',
 						description: 'Send answer to callback query sent from inline bot',
+						action: 'Answer Inline Query a callback',
```

Comment:
Answer Inline Query callback

Reply:
Changed to `Answer an inline query callback`